setup.py

```diff
@@ -1,44 +0,0 @@
-from setuptools import setup, find_packages
```

Comment:
Can we keep a generic setup.py that somehow "auto read" the ppyproject.toml ?

Reply:
I think pip will prefer `setup.py` over `pyproject.toml` if we do this during `pip install .`. I suggest we remove `setup.py` completely with this PR.

Reply:
I'll add a comment on the Poetry PR related to this request as I discovered a potential issue with the current plan.

Reply:
You won't lose any pip functionality with this change. In fact, you can use your old pip commands if you wish to do so even if/when we migrate fully to poetry.